### PR TITLE
JSdoc build fixes and move build script out of the root folder

### PR DESCRIPTION
**Pull request by juliusknorr**

### CODE REVIEW

**Review by skjnldsv — APPROVED** [review]

File: build/buildjsdocs.sh [hunk 1]
```diff
@@ -8,10 +8,9 @@
 # @copyright 2014 Vincent Petry <pvince81@owncloud.com>
```
Comment (MorrisJobke) [inline build/buildjsdocs.sh]: When renaming this file you also need to remove it from the CI check for files in the root folder ;)

**Review by MorrisJobke — APPROVED** [review]
👍

File: build/buildjsdocs.sh (hunk 1 above)
Comment (MorrisJobke) [reply build/buildjsdocs.sh]: And see https://drone.nextcloud.com/nextcloud/server/11707/95

**Comment by rullzer** [comment]
Conflicts

**Comment by juliushaertl** [comment]
Conflicts resolved.

**Pull request merged**